src/components/VersionChooser/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Col, Form, Row } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import VersionSelect, { Version } from '../VersionSelect';
+import { gettext } from '../../utils';
+import styles from './styles.module.scss';
+
+type PublicProps = {
+  versions: Version[];
+};
+
+class VersionChooserBase extends React.Component<PublicProps> {
+  render() {
+    const { versions } = this.props;
+
+    const listedVersions = versions.filter(
+      (version) => version.channel === 'listed',
+    );
+    const unlistedVersions = versions.filter(
+      (version) => version.channel === 'unlisted',
+    );
+
+    return (
+      <div className={styles.VersionChooser}>
+        <Form>
+          <Row className={styles.heading}>
+            <Col>
+              <h3>{gettext('Compare changes between two versions')}</h3>
```

Comment:
You could simplify this prompt to *Compare changes* since the labels you added, *Choose an old version* and *Choose a new version*, imply enough context.